src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Main {
```

Comment:
don't commit this file, you can keep it locally ,but it shouldn't be in the PR

Reply:
not fixed